nilearn/datasets/atlas.py

```diff
@@ -199,7 +193,8 @@ def fetch_atlas_craddock_2012(data_dir=None, url=None, resume=True, verbose=1):
 
 def fetch_atlas_destrieux_2009(lateralized=True, data_dir=None, url=None,
                                resume=True, verbose=1):
-    """Download and load the Destrieux cortical atlas (dated 2009) [1]_ [2]_.
+    """Download and load the Destrieux cortical atlas (dated 2009) 
+    :footcite:`Fischl2004Automatically` :footcite:`Destrieux2009sulcal`.
```

Comment:
Could you have the references on the next line to avoid weird rendering on the API page?
Also, you should be able to specify multiple keys at one.
Something like this maybe?:

```python
def fetch(args):
    """Download and load the Destrieux cortical atlas (dated 2009).

        See :footcite:`Fischl2004Automatically,Destrieux2009sulcal`.

        Parameters
        -------
    """
```